modules/openapi-generator/src/main/resources/dart2/api_helper.mustache

```diff
@@ -58,7 +59,7 @@ String parameterToString(dynamic value) {
       {{/isEnum}}
     {{/model}}
   {{/models}}
-  return value.toString();
+  return '$value';
```

Comment:
if `parameterToString` is allowed to return null, then `return value?.toString();` 
if not, then make it `return value?.toString() ?? '';`

Reply:
This function does not return null, always a `String` which may be an empty one.
Also, we cannot use `value?` since the template must work with Dart 2.0 – no support for this operator yet.